index.js

```diff
@@ -62,8 +62,8 @@ function generateBarChart(percent, size) {
 
   const frac = (size * 8 * percent) / 100;
   const barsFull = Math.floor(frac / 8);
-  const semi = frac % 8;
-  const barsEmpty = size - barsFull - 1;
+  const semi = frac % 8 || 8;
+  const barsEmpty = Math.max(size - barsFull - 1, 0);
```

Comment:
don't think these lines need to be touched

Reply:
@casperdcl take `percent = 100`. we will have `semi = 0` but it should be `8` in this case. also `barsEmpty` will be equal to `-1` which will throw an error here `repeat(barsEmpty)`